pooch/tests/test_downloaders.py

```diff
@@ -495,6 +500,7 @@ def test_api_version(self, httpserver, api_version, api_response):
         "api_version, api_response",
         [("legacy", legacy_api_response), ("new", new_api_response)],
     )
+    @pytest.mark.network
```

Comment:
Same case with this test, which also makes use of `httpserver`. Do you face any problem if you run this test without connection?
```suggestion
```

Reply:
Same case, same answer. ;)